### PR TITLE
Add new 'tags' for cards that heal horror and heal damage

### PR DESCRIPTION
**Pull request by zzorba**
Similar to traits (hh., hd. ,but more to come if more weird conditions happen.

### CODE REVIEW

File: pack/dwl/tmm.json [hunk 1]
```diff
@@ -203,6 +204,7 @@
         "position": 116,
         "quantity": 2,
         "skill_willpower": 1,
+        "tags": "hh.",
         "text": "Uses (3 supplies).\n[free]Spend 1 supply, exhaust Smoking Pipe, and take 1 damage: Heal 1 horror.",
```
Comment (drawntotheflame) [inline pack/dwl/tmm.json:208]:
```suggestion
        "text": "Uses (3 supplies).\n[free] Spend 1 supply, exhaust Smoking Pipe, and take 1 damage: Heal 1 horror.",
```

File: pack/dwl/tmm.json [hunk 2]
```diff
@@ -220,6 +222,7 @@
         "position": 117,
         "quantity": 2,
         "skill_willpower": 1,
+        "tags": "hd.",
         "text": "Uses (3 supplies).\n[free]Spend 1 supply, exhaust Painkillers, and take 1 horror: Heal 1 damage.",
```
Comment (drawntotheflame) [inline pack/dwl/tmm.json:226]:
```suggestion
        "text": "Uses (3 supplies).\n[free] Spend 1 supply, exhaust Painkillers, and take 1 horror: Heal 1 damage.",
```

File: pack/eoe/eoep.json [hunk 3]
```diff
@@ -932,6 +933,7 @@
         "skill_willpower": 2,
         "slot": "Arcane",
         "subname": "Gateway to Paradise",
+        "tags": "hd.hh.",
```
Comment (drawntotheflame) [inline pack/eoe/eoep.json:936]: Does the order 'hh.hd.' or 'hd.hh.' matter?

File: pack/promo/tftbw.json [hunk 4]
```diff
@@ -16,6 +16,7 @@
                 "text": [
                     "[Hh]eals? (that much )?(((\\d+|all|(X total)) )?damage (from that asset )?(and|or) )?((\\d+|all|(X total)) )?horror"
```
Comment (drawntotheflame) [inline pack/promo/tftbw.json:17]: will we delete the 'text' line once the tag is up and running?

File: pack/tcu/tcu.json [hunk 5]
```diff
@@ -16,6 +16,7 @@
                 "text": [
                     "[Hh]eals? (that much )?(((\\d+|all|(X total)) )?damage (from that asset )?(and|or) )?((\\d+|all|(X total)) )?horror"
```
Comment (drawntotheflame) [inline pack/tcu/tcu.json:17]: Same q regarding deleting the 'text' line.

File: pack/core/core.json [hunk 6]
```diff
@@ -424,6 +424,7 @@
         "position": 19,
         "quantity": 1,
         "skill_willpower": 1,
+        "tags": "hh.hd.",
```
Comment (drawntotheflame) [inline pack/core/core.json:427]: Later you switch to the alphabetical 'hd.hh.' so I think change here too: 

```suggestion
        "tags": "hd.hh.",
```

File: pack/tfa/tbb.json [hunk 7]
```diff
@@ -10,6 +10,7 @@
         "position": 149,
         "quantity": 2,
         "skill_willpower": 1,
+        "tags": "hd.",
         "text": "Play only as your first action.<br/>Heal 1 damage (2 instead if you drew a treachery this round). Then, draw 1 card.",
```
Comment (drawntotheflame) [inline pack/tfa/tbb.json:14]:
```suggestion
        "text": "Play only as your first action.\nHeal 1 damage (2 instead if you drew a treachery this round). Then, draw 1 card.",
```

**Review by drawntotheflame — COMMENTED** [review]
Great job!

File: pack/promo/tftbw.json (hunk 4 above)
Comment (zzorba) [reply pack/promo/tftbw.json:17]: Eventually yeah, I wanted to leave it in place for people who were expecting the API.

The website will now ignores the 'text' field going forward though, so its value is only there for legacy api users.

**Comment by zzorba** [comment]
This can be merged without the corresponding arkhamdb change, but the card import would need to be re-run after the arkhamdb deploy to make sure the new fields are picked up.

**Pull request merged**